app/models/katello/content_view.rb

```diff
@@ -494,7 +494,7 @@ def create_new_version(major = next_version, minor = 0, components = self.compon
                                            :components => components
                                           )
 
-      increment!(:next_version) if minor == 0
+      next_version = major + 1
```

Comment:
I'm not able to publish after this change:

```
no implicit conversion of Integer into String (TypeError)                            
 | /home/vagrant/katello/app/models/katello/content_view.rb:497:in `+'                               
 | /home/vagrant/katello/app/models/katello/content_view.rb:497:in `create_new_version'                 
 | /home/vagrant/katello/app/lib/actions/katello/content_view/publish.rb:13:in `plan'         
```

Reply:
this seems to help:

```ruby
self.next_version = major.to_i + 1
```

However I'm not sure how to get the change to persist. The old `increment!` method would auto-save, and I'm not sure where the correct place to do the save is now. We can look at it more tomorrow.